sys/net/gnrc/network_layer/ipv6/nib/_nib-router.c

```diff
@@ -191,11 +191,11 @@ static gnrc_pktsnip_t *_build_ext_opts(gnrc_netif_t *netif,
             }
         }
     }
-    ltime = (gnrc_netif_is_6lbr(netif)) ?
-            (SIXLOWPAN_ND_OPT_ABR_LTIME_DEFAULT) :
-            (abr->valid_until - evtimer_now_min());
-    (void)ltime;    /* gnrc_sixlowpan_nd_opt_abr_build might evaluate to NOP */
-    abro = gnrc_sixlowpan_nd_opt_abr_build(abr->version, ltime, &abr->addr,
+    ltime_min = (gnrc_netif_is_6lbr(netif)) ?
+                (SIXLOWPAN_ND_OPT_ABR_LTIME_DEFAULT) :
+                ((abr->valid_until_ms - evtimer_now_msec()) / MS_PER_SEC / SEC_PER_MIN);
```

Comment:
even more readable without the superfluous parens 

```suggestion
    ltime_min = gnrc_netif_is_6lbr(netif)
              ? SIXLOWPAN_ND_OPT_ABR_LTIME_DEFAULT
              : gnrc_ipv6_nib_abr_valid_offset(abr);
```